reactor-netty-http/src/test/java/reactor/netty/http/server/HttpServerOutboundCompleteTest.java

```diff
@@ -282,28 +297,44 @@ void httpPipeliningGetRespondsSendMono(boolean enableAccessLog) throws Exception
 			assertThat(recorder.onCompleteIsReceived.get()).isEqualTo(16);
 			assertThat(recorder.onTerminateIsReceived.get()).isEqualTo(16);
 
-			if (enableAccessLog) {
+			if (enableMetricsAndAccessLog) {
 				assertThat(logTracker.latch.await(5, TimeUnit.SECONDS)).isTrue();
 				assertThat(logTracker.actualMessages).hasSize(16);
 			}
 		}
 		finally {
+			if (registry != null) {
+				for (int i = 0; i < 16; i++) {
+					assertTimer(registry, "reactor.netty.http.server.response.time", METHOD, "GET", STATUS, "200", URI, "/" + i).isNotNull();
```

Comment:
```suggestion
					assertTimer(registry, HTTP_SERVER_PREFIX + RESPONSE_TIME, METHOD, "GET", STATUS, "200", URI, "/" + i).isNotNull();
```

then the HTTP_SERVER_PREFIX and RESPONSE_TIME constants must also be imported:

import static reactor.netty.Metrics.HTTP_SERVER_PREFIX;
import static reactor.netty.Metrics.RESPONSE_TIME;

Reply:
I applied the suggestion